pom.xml

```diff
@@ -1104,7 +1140,7 @@
       <id>wildflydev</id>
       <properties>
         <jboss.home>${server.home}</jboss.home>
-        <jboss.version>${wildfly.dev.version}</jboss.version>
+        <jboss.version>${wildfly.version}</jboss.version>
```

Comment:
what is the purpose of this profile after this change? it is the same as the one above now, can we get rid of it then?

Reply:
We actually could rid of the profile, I was just trying to keep the change simple. However, you're right that it should probably just go away.

Reply:
I removed the profile

Reply:
the profile in root pomx.ml was in pair with https://github.com/jbossws/jbossws-cxf/blob/main/modules/testsuite/pom.xml#L667-L667 - that one seems to be also duplicated so I would say we can remove too

Reply:
Oh, good catch. I'll remove this one too and the profile in the wildfly-build.yml.